proto/internal/impl/v1/impl.proto

```diff
@@ -33,6 +33,10 @@ message DecodedZedToken {
   }
   message V1ZedToken {
     string revision = 1;
+
+    // datastore_unique_id_prefix is a prefix of the unique ID for the datastore that created
```

Comment:
why is this a prefix and not the entire string? i don't understand

can you write down here examples of the uniqueIDs per datastore?

Reply:
It was deemed too big; see the discussion above

Reply:
Yes I saw that but i'm suggesting leaving a comment in the code with the conclusion